pkg/apis/sessions/session_state.go

```diff
@@ -97,6 +101,8 @@ func (s *SessionState) GetClaim(claim string) []string {
 		return groups
 	case "preferred_username":
 		return []string{s.PreferredUsername}
+	case "introspect-claims":
```

Comment:
Other options here use an underscore, we should be consistent

Reply:
sure will update it